Makefile

```diff
@@ -194,7 +194,7 @@ pylint: $(PYSOURCES) $(wildcard tests/*.py)
 		setup.py khmer/[!_]*.py khmer/__init__.py scripts/*.py tests \
 		oxli/*.py || true
 
-pylint_report.txt: ${PYSOURCES} $(wildcard tests/*.py)
+pylint_report.txt: ${PYSOURCES} $(wildcard tests/*.py) $(wildcard sandbox/*.py)
```

Comment:
Yes, otherwise `make diff_pylint_report.txt` won't pick up changes made to `sandbox/*.py` scripts since the last run.